### PR TITLE
LOC CHECKIN | dotnet/winforms master | 20190401

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.ko.xlf [hunk 1]
```diff
@@ -74,7 +74,6 @@
       </trans-unit>
       <trans-unit id="ColorEditorStandardTab">
         <source>Web</source>
-        <target state="needs-review-translation">Web</target>
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.ko.xlf]: example of a removed target

**Comment by janbaltus** [comment]
@zsd4yr   We added target tag for missing instances and updated PR

File: src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.ko.xlf [hunk 2]
```diff
@@ -74,7 +74,7 @@
...
+	<target state="new">Web</target>
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.ko.xlf]: @v-zbsail the tabbing is incorrect in some places now

Comment (janbaltus) [reply src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.ko.xlf]: @zsd4yr we used space instead of tab and updated PR

Comment (zsd4yr) [reply src/System.Windows.Forms.Design.Editors/src/Resources/xlf/SR.ko.xlf]: thanks 😄

**Pull request merged**